core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -110,14 +111,17 @@ public int getUsedSpacePercent() {
   private static final long serialVersionUID = 2335205655766736309L;
 
   private final transient MasterProcess mMasterProcess;
+  private final transient MetaMaster mMetaMaster;
 
   /**
    * Creates a new instance of {@link WebInterfaceGeneralServlet}.
    *
    * @param masterProcess Alluxio master process
+   * @param metaMaster Alluxio meta master
    */
-  public WebInterfaceGeneralServlet(MasterProcess masterProcess) {
+  public WebInterfaceGeneralServlet(MasterProcess masterProcess, MetaMaster metaMaster) {
     mMasterProcess = masterProcess;
+    mMetaMaster = metaMaster;
```

Comment:
this is minor, but do we need to pass metamaster here? can't we just 
`mMetaMaster  = mMasterProcess.getMaster(MetaMaster.class);`?

Reply:
Changed, thanks!